guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -48,8 +48,14 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
-. Enter the details for how you want the registered host to be configured.
+. Enter the details for how you want the registered host to be configured. Note:
```

Comment:
I'd drop "Note:". alternative:

```suggestion
. Enter the details for how you want the registered host to be configured.
+
[NOTE]
====
...
====
```

Reply:
```suggestion
. Enter the details for how you want the registered host to be configured.
```
An alternative. I think the bulleted list is "noteable" enough.